src/views/DatacenterBrowser/Progress.js

```diff
@@ -1,9 +1,15 @@
 import m from "mithril";
+import moment from "moment";
```

Comment:
should we still be bringing in moment if we're not using it now?

Reply:
this should be fixed in 478b105